Makefile

```diff
@@ -73,12 +74,12 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
-format: ## format Jsonnet sources
-	bin/jsonnetfmt -i $(sources)
+format: ## format Jsonnet sources and libraries
+	bin/jsonnetfmt -i $(sources) $(libraries)
 .PHONY: format
 
-lint: ## lint Jsonnet sources
-	bin/jsonnet-lint $(sources)
+lint: ## lint Jsonnet sources and libraries
+	bin/jsonnet-lint $(sources) $(libraries)
```

Comment:
Ditto.

Reply:
Thanks) The circle ci uses the make file, so this is now taken into account.